clang/lib/StaticAnalyzer/Core/SValBuilder.cpp

```diff
@@ -151,10 +151,10 @@ SValBuilder::getRegionValueSymbolVal(const TypedValueRegion *region) {
   return nonloc::SymbolVal(sym);
 }
 
-DefinedOrUnknownSVal SValBuilder::conjureSymbolVal(const void *SymbolTag,
```

Comment:
This overload would have been removed, but its behaviour is not entirely the same as obtaining the type from the expression directly. (See the check for `isGLValue`). Without complicating the refactor in this PR I decided to leave it as is, and add a new parameter for the CFG Element Ref